common/src/View/EntityAttributeModel.cpp

```diff
@@ -251,34 +252,49 @@ namespace TrenchBroom {
             updateFromMapDocument();
         }
 
-        static std::vector<AttributeRow> buildVec(const std::map<std::string, AttributeRow>& rows) {
-            std::vector<AttributeRow> result;
-            result.reserve(rows.size());
-            for (auto& [key, row] : rows) {
-                unused(key);
-                result.push_back(row);
+        static auto makeNameToAttributeRowMap(const std::vector<AttributeRow>& rows) {
+            std::map<std::string, AttributeRow> result;
+            for (const auto& row : rows) {
+                result[row.name()] = row;
             }
             return result;
         }
 
-        /* FIXME: remove unused code
-        static auto buildAttributeToRowIndexMap(const std::vector<AttributeRow>& rows) {
-            std::map<std::string, int> result;
-            for (size_t i = 0; i < rows.size(); ++i) {
-                const AttributeRow& row = rows[i];
-                result[row.name()] = static_cast<int>(i);
+        using AttributeRowMap = std::map<std::string, AttributeRow>;
```

Comment:
We use the order of these maps when inserting rows, so I'd prefer to stick with a sorted map, since it'd potentially make reproducing bugs more difficult if it was unsorted. If we had a kdl::sorted_map that would probably be ideal.